plugins/ScheduledReports/ScheduledReports.php

```diff
@@ -207,6 +200,18 @@ public function getReportMetadata(&$reportMetadata, $reportType, $idSite)
         $reportMetadata = $filteredReportMetadata;
     }
 
+    private function shouldReportBeIgnored($reportMetadata)
+    {
+        // Some plugins may not define report categories
+        if(empty($reportMetadata['category'])) {
+            return false;
+        }
+
+        $isAPI = $reportMetadata['category'] == 'API';
+        $isMultiSitesGetOne = $reportMetadata['category'] == Piwik::translate('General_MultiSitesSummary') && $reportMetadata['name'] == Piwik::translate('General_SingleWebsitesDashboard');
```

Comment:
ideally this would be only executed if needed as we otherwise translate keys loads of times which has been a bit "slow" in the past but is also fine like this if needed